app/src/main/java/org/oppia/android/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt

```diff
@@ -1,29 +1,21 @@
 package org.oppia.android.app.administratorcontrols.administratorcontrolsitemviewmodel
 
-import android.content.Intent
-import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.Fragment
-import org.oppia.android.R
-import org.oppia.android.app.shim.IntentFactoryShim
+import androidx.lifecycle.ViewModel
+import org.oppia.android.app.administratorcontrols.AdministratorControlsFragment
```

Comment:
It looks like a unused import (ktlint might have ignored it as it is used in the KDoc)

Reply:
What's to be done here?

Reply:
Removed them

Reply:
Thanks, you can resolve this now